drivers/platform/x86/Kconfig

```diff
@@ -561,6 +561,21 @@ config EEEPC_WMI
 	  If you have an ACPI-WMI compatible Eee PC laptop (>= 1000), say Y or M
 	  here.
 
+config ASUS_WRC
+	tristate "ASUS Wireless Radio Control Driver"
+	depends on ACPI
+	depends on INPUT
+	select INPUT_SPARSEKMAP
+	---help---
+	  This is a driver for the wireless radio control (airplane mode hotkey)
+	  found on newer Asus notebooks, also known as ATK4002 or ASHS.
+
```

Comment:
I was following the language in ASUS_NB_WMI, and I think they use that language there to differentiate from the "older" laptops handled by ASUS_LAPTOP. I agree it is not precise in any manner, but if we use the model name would this description be updated to cover all modules using this interface? Maybe we should say "some Asus laptops" instead, what do you think?